content/posts/0024-bionic-builder-unsafe.md

```diff
@@ -0,0 +1,90 @@
+---
+title: Paketo's Bionic Builder Is Unsafe
```

Comment:
I'd try to focus on the fact that some other projects are still defaulting to the EOL builder, rather than calling out that our builder is unsafe.

```suggestion
title: Important note: Many projects still use the Bionic Builder that has surpassed it's end-of-life
```

Likely too long for a title, but still shows the direction I would rather focus on.

Reply:
I wanted a little bit of click-bait with the title. Something sensational to get people to read it. I don't think it's necessarily wrong though. Using something that hasn't been updated in six months is unsafe in today's world.

That said, I understand that we have a brand to maintain too. If folks feel this is over the top, I can dial it back a little.

Channeling my inner news reporter, a few other ideas to get attention:

- Please Stop Using Paketo's Bionic Builder
- Uh-Oh, You May Be Accidentally Using the EOL Bionic Builder
- Check Your Builder! It May No Longer Be Receiving Updates

Thoughts?

Reply:
I do not have a strong opinion on what direction we go with for this title. I would say that unless @loewenstein has a strong suggestion, I would just pick a title and run it. There is nothing stopping us from changing the title.

Reply:
No strong opinion. Feel free to proceed.